src/base-commands/base-command.js

```diff
@@ -55,7 +57,8 @@ class BaseCommand extends Command {
 
   async catch(error) {
     if (!this.logger || instanceOf(error, CLIError)) {
-      return super.catch(error);
+      super.catch(error);
+      return;
```

Comment:
Is this change necessary? Seems safer to return whatever super returns, if anything.